docs/build_profiles.md

```diff
@@ -0,0 +1,19 @@
+# Build Profiles
+Mbed 5.0 supports three primary build profiles, *default*, *debug* and *small*. When using
+the online compiler the *default* profile is used. When building from the command line
+the desired profile can be can be selected by adding the ```--profile mbed-os/tools/<profile>``` command line flag.
```

Comment:
should we also or just use --profile <profile> as it's much user-friendly than path_to_profiles/<profile> ?